pages/app/index.html

```diff
@@ -1,38 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cloudscape demo pages</title>
 
-<head>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Cloudscape demo pages</title>
+    <script>
+      const isDevelopment = '<%= process.env.NODE_ENV %>' !== 'production';
+      const developmentCsp = "connect-src 'self' ws://" + location.host + ';';
+      const safeCsp = "default-src 'self'; style-src 'self'; font-src data:;" + (isDevelopment ? developmentCsp : '');
+      const themingCsp = "default-src 'self'; style-src 'unsafe-inline' 'self'; font-src data:;";
+      // Custom CSP required for how we load SVGs in CodeEditor component
+      //unsafe-inline is required for the react-resizable box in the code-editor
+      const codeEditorCsp =
+        "default-src 'self'; style-src 'unsafe-inline' 'self'; font-src data:;  img-src 'self' data:;" +
+        (isDevelopment ? developmentCsp : '');
 
-  <script>
-    const isDevelopment = "<%= process.env.NODE_ENV %>" !== "production";
-    const developmentCsp = "connect-src 'self' ws://" + location.host + ";" ;
-    const safeCsp = "default-src 'self'; style-src 'self'; font-src data:;" + (isDevelopment ? developmentCsp : "");
-    const themingCsp = "default-src 'self'; style-src 'unsafe-inline' 'self'; font-src data:;"
-    // Custom CSP required for how we load SVGs in CodeEditor component
-    //unsafe-inline is required for the react-resizable box in the code-editor
-    const codeEditorCsp = "default-src 'self'; style-src 'unsafe-inline' 'self'; font-src data:;  img-src 'self' data:;" + (isDevelopment ? developmentCsp : "");
- 
-    let csp;
-    if (location.hash.indexOf('code-editor') > -1) {
-      csp = codeEditorCsp;
-    } else if (location.hash.indexOf('theming') > -1) {
-      csp = themingCsp;
-    } else {
-      csp = safeCsp;
-    }
-    document.write('<meta http-equiv="Content-Security-Policy" content="' + csp + '" />');
-  </script>
-</head>
+      let csp;
+      if (location.hash.indexOf('code-editor') > -1) {
+        csp = codeEditorCsp;
+      } else if (location.hash.indexOf('theming') > -1) {
+        csp = themingCsp;
+      } else {
+        csp = safeCsp;
+      }
+      document.write('<meta http-equiv="Content-Security-Policy" content="' + csp + '" />');
 
-<body>
-  <!-- #b element should be there for compatibility with global navigation -->
-  <div id="b">
-    <div id="app"></div>
-  </div>
-</body>
+      window.panorama = function (functionName, opts) {
+        console.log(functionName, opts);
+      };
```

Comment:
The only real change in this file. The rest was auto formatted.

Reply:
Can we avoid changing this file? This script has to be inline, because it is the only supported way to configure CSP header.

For everything else it is better to use separate files.

Reply:
We can avoid it if it can possibly cause problems. Will update PR.

Reply:
P.S. I am also not sure if about losing test coverage for our components behavior when `window.panorama` is undefined (which is a very common case).

Can we conditionally apply panorama mock only in particular scenarios?